websites/H/HVNC/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "941317056589086730"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const inPath = document.location.pathname,
+		select = document.querySelector,
```

Comment:
Lets not use this _function_ and call document each time

Reply:
done, thanks